INSTALL.md

```diff
@@ -12,6 +12,16 @@ source.
     $ make
     $ sudo make install
 
+On RHEL, CentOS and Fedora machines, following commands will build and install uftrace from source. 
+
+    $ sudo yum install elfutils-devel	        # for debug info       (optional)
+    $ sudo yum install pandoc		            # for man pages        (optional)(Fedora only)
+    $ sudo yum install python2-devel	        # for python scripting (optional)
+    $ sudo yum install ncurses-devel	        # for TUI              (optional)
+    $ sudo yum install pkgconfig	            # for configure        (optional)(Fedora use pkgconf-pkg-config)
```

Comment:
Could please align the comments so that they can start from the same position?  Also I'd prefer saying single parenthesis separated by comma like "(optional, Fedora only)".

Reply:
@namhyung Thanks. I fixed it.